projects/plugins/jetpack/_inc/client/search/dashboard/module-control.jsx

```diff
@@ -39,6 +40,8 @@ const INSTANT_SEARCH_DESCRIPTION = __(
 	'jetpack'
 );
 const SEARCH_SUPPORT = __( 'Search supports many customizations. ', 'jetpack' );
+const SEARCH_CUSTOMIZE_URL = 'customize.php?autofocus[section]=jetpack_search';
+const WIDGETS_EDITOR_URL = 'customize.php?autofocus[panel]=widgets';
```

Comment:
Another approach here might be to use the Edit Widgets page at `/wp-admin/widgets.php`.

Pro: Page weight is significantly lighter.
Con: No live preview.

Reply:
I prefer the customizer. More modern (than widgets.php), live preview and return URL :-)
What do you think @bluefuton

Reply:
I prefer the Customizer for the moment. The old drag and drop UI is pretty poor at /wp-admin/widgets.php I think, especially when there are lots of widgets available. The new block-based widget editor will be launched at /wp-admin/widgets.php in WordPress 5.8, so let's switch to that when it's available 🙂